### PR TITLE
br: reset cloud_admin and root after ebs restoration

### PR DESCRIPTION
**Pull request by 3pointer**
### What problem does this PR solve?

Issue Number: close https://github.com/pingcap/tidb/issues/40418

Problem Summary:
1. For ebs volume-snapshot restore, we need to reset some accounts otherwise db_initializer will fail due to accounts existing.

### What is changed and how it works?
1. add a new config `reset-sys-users`. in case we need to reset more users in future.
2. remove cloud_admin by default
3. reset the root password and let cloud_admin control it.
### Check List

Tests 

- [x] Unit test
- [ ] Integration test
- [x] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: br/pkg/restore/systable_restore.go [hunk 1]
```diff
@@ -71,6 +75,72 @@ func isStatsTable(tableName string) bool {
...
+		// SQLs here only contain table name and database name, seems it is no need to redact them.
+		if err := rc.db.se.Execute(ctx, sql); err != nil {
+			log.Warn("failed to execute SQL restore system database",
+				zap.Stringer("database", db.Name),
+				zap.String("sql", sql),
+				zap.Error(err),
+			)
+			return berrors.ErrUnknown.Wrap(err).GenWithStack("failed to execute %s", sql)
+		}
+		log.Info("successfully restore system database",
```
Comment (WangLe1321) [inline br/pkg/restore/systable_restore.go:99]: I think this log can be more clear, for example "successfully execute SQL to clear system user". So is the error log.

**Review by fengou1 — APPROVED** [review]
good enough to ship it

File: br/pkg/restore/systable_restore.go (hunk 1 above)
Comment (fengou1) [reply br/pkg/restore/systable_restore.go]: fixed in https://github.com/pingcap/tidb/pull/39986/commits/a1f401fc4c2894aa7086ca3e2270c30d65453fa2

**Review by joccau — APPROVED** [review]
LGTM

**Review by WangLe1321 — APPROVED** [review]
LGTM

**Pull request merged**